aws-xray-sdk-python-master/sample-apps/flask/application.py

```diff
@@ -64,4 +64,4 @@ def default():
         port = '5000'
     else:
         host, port = address.split(":")
-    app.run(host=host, port=int(port), debug=True)
+    app.run(host=host, port=int(port), debug=False)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Don&#39;t hardcode the `DEBUG` variable because some system extensions might be configured to use a diffrent value. Instead, use OS environment variables to set up the debug feature.